qiskit_experiments/framework/base_experiment.py

```diff
@@ -265,31 +265,52 @@ def _finalize(self):
         """
         pass
 
-    def _run_jobs(self, circuits: List[QuantumCircuit], **run_options) -> List[Job]:
-        """Run circuits on backend as 1 or more jobs."""
+    def _max_circuits(self):
+        """Calculate the maximum number of circuits per job for the experiment."""
         # Get max circuits for job splitting
         max_circuits_option = getattr(self.experiment_options, "max_circuits", None)
         max_circuits_backend = self._backend_data.max_circuits
+
         if max_circuits_option and max_circuits_backend:
-            max_circuits = min(max_circuits_option, max_circuits_backend)
+            return min(max_circuits_option, max_circuits_backend)
         elif max_circuits_option:
-            max_circuits = max_circuits_option
+            return max_circuits_option
         else:
-            max_circuits = max_circuits_backend
+            return max_circuits_backend
+
+    def job_info(self, backend=None):
+        """Get information about job distribution for the experiment on a specific backend."""
```

Comment:
This is a user-facing docstring so it needs to describe the input and output parameters. Consult other docstrings for the style.